include/swift/AST/SemanticAttrs.h

```diff
@@ -23,7 +23,8 @@
 
 namespace swift {
 namespace semantics {
-#define SEMANTICS_ATTR(NAME, C_STR) constexpr static const StringLiteral NAME = C_STR;
+#define SEMANTICS_ATTR(NAME, C_STR) \
+  constexpr static const llvm::StringLiteral NAME = C_STR;
```

Comment:
StringLiteral is injected into the swift namespace by https://github.com/apple/swift/blob/master/include/swift/Basic/LLVM.h. Maybe just include that and get rid of llvm::?